alacritty_terminal/src/tty/unix.rs

```diff
@@ -158,12 +158,14 @@ impl ShellUser {
 }
 
 #[cfg(not(target_os = "macos"))]
-fn default_shell_command(shell: &str, _user: &str) -> Command {
+fn default_shell_command(shell: &str, _user: &str, _home: &str) -> Command {
     Command::new(shell)
 }
 
 #[cfg(target_os = "macos")]
-fn default_shell_command(shell: &str, user: &str) -> Command {
+fn default_shell_command(shell: &str, user: &str, home: &str) -> Command {
+    use std::path::Path;
```

Comment:
Should be moved to the top with a cfg, so the cfg can be easily removed if other platforms start relying on this import.

Reply:
🫡